doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -92,6 +92,16 @@ to affect how Pacemaker calls them.
    +------------------+---------------+-----------------------------------------------------+
    | Meta-Attribute   | Default       | Description                                         |
    +==================+===============+=====================================================+
+   | enabled          | true          | .. index::                                          |
+   |                  |               |    single: alert; meta-attribute, enabled           |
+   |                  |               |    single: meta-attribute; enabled (alert)          |
+   |                  |               |    single: enabled; alert meta-attribute            |
+   |                  |               |                                                     |
+   |                  |               | If false for an alert agent, the alert agent will   |
+   |                  |               | not be used. If true for an alert agent and false   |
+   |                  |               | for a particular recipient of that agent, that      |
+   |                  |               | recipient will not be used.                         |
```

Comment:
I would say just "alert" instead of "alert agent" (you could have multiple alerts with the same agent for various reasons)

Reply:
I totally agree. I was trying to be consistent with the rest of this section, which says "alert agents" when it means alerts. I can update the rest where appropriate.

Reply:
updated